lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -96,6 +107,31 @@ def default_collation=(collation)
             @default_collation = collation
           end
 
+          def default_text_limit=(text_limit)
+            text_limit.nil? or text_limit.is_a?(Integer) or raise ArgumentError, "text limit must be an integer or nil (got #{text_limit.inspect})"
+            @default_text_limit = text_limit
+          end
+
+          def default_string_limit=(string_limit)
+            string_limit.nil? or string_limit.is_a?(Integer) or raise ArgumentError, "string limit must be an integer or nil (got #{string_limit.inspect})"
+            @default_string_limit = string_limit
+          end
+
+          def default_null=(null)
+            [true, false, nil].include? null or raise ArgumentError, "null must be either true, false, or nil (got #{null.inspect})"
```

Comment:
We use parens these days, so that'd be `include?(null)`.

Ditto below in a few places.